packages/engine/src/redux/__tests__/selectors.test.ts

```diff
@@ -62,14 +61,11 @@ describe("getNextNonce", () => {
     libraryAddress: "0x0",
     ourIndex: 0,
     participants: ["0x0", "0x0"],
-    channelNonce: 0,
+    channelNonce: "0",
```

Comment:
Why is this a string? If it is because `ChannelNonce` is a `uint256` (I forget if it is, I guess it is), then you should use `"0x00"`